scijava/scijava-ops-api/src/main/java/org/scijava/ops/api/OpBuilder.java

```diff
@@ -276,22 +313,38 @@ public Type getType() {
 			return env.op(opName, specialType, new Nil<?>[0], outType, hints);
 		}
 
+		/**
+		 * Match a {@link org.scijava.function.Computers} op to work with pre-allocated output.
+		 */
 		public Computers.Arity0<O> computer() {
 			return matchComputer(env, opName, outType);
 		}
 
-		public O create() {
-			return producer().create();
-		}
-
+		/**
+		 * As {@link #computer} but using the provided {@code Hints}.
+		 */
 		public Computers.Arity0<O> computer(Hints hints) {
 			return matchComputer(env, opName, outType, hints);
 		}
 
+		/**
+		 * Directly run this {@link org.scijava.function.Producer} op and get its output.
+		 *
+		 * @see #producer
+		 * @return The {@code O} created by this op
+		 */
+		public O create() {
+			return producer().create();
+		}
+
+		/**
+		 * As {@link #create} but using the provided {@code Hints}.
```

Comment:
These `Hints` signatures weren't part of my original design, and I think we can do better. We could have a separate `hints(Hints)` call in the chain that associates the hints into the build, so that we don't have to propagate twice as many method signatures over all the builder classes. Or it could maybe just go into the original `op(String)` function, as a second `op(String, Hints)` right away. What do you think?

Reply:
I definitely am in favor of a second `op(String, Hints)` signature! I'll make the change